draft-ietf-masque-h3-datagram.md

```diff
@@ -356,6 +359,10 @@ codes 204 (No Content), 205 (Reset Content), and 206 (Partial Content) MUST NOT
 be sent on responses that use the Capsule Protocol. A receiver that observes a
 violation of these requirements MUST treat the HTTP message as malformed.
 
+Receivers of capsules can read them from the data stream in a streaming fashion.
```

Comment:
This reads a bit oddly — it seems to imply that receivers could choose to read them in some non-streaming fashion. What would that be? What is a streaming fashion, really?

Reply:
Do you mean reading partial capsules?

Reply:
The idea is that an application might read capsule headers and then try to read the payload in a single lump, rather that stream it out to a memory buffer or straight to another IO destination.

Reading things as they arrive might be obvious to some implementers but I was trying to point out the approach to help the other points made

Reply:
Yeah that's the intent. Maybe this text should mirror more closely what we have below in {{datagram-capsule}} since that's clearer

Reply:
Yeah, the second instance of this text was clearer — a forward reference sounds better.

Reply:
the problem is that layer text is specific to DATAGRAM capsule, and here we talk about all of them, Let me try a little switcheroo I think might work